content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -74,16 +77,17 @@ In order to deploy Kubeflow on your existing Amazon EKS cluster, you need to pro
 
 1. Run the following commands to set up your environment and initialize the cluster.
 
-    ```shell
-    kfctl init ${KFAPP} --config=${CONFIG} -V
-    cd ${KFAPP}
-
-    kfctl generate all -V
-    kfctl apply all -V
     ```
-
-    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`. It will be used as eks cluster name.
-    * CONFIG - Path to the configuration file
+    # Set the path to the base directory where you want to store one or more 
+    # Kubeflow deployments. For example, /opt/.
+    # Then set the Kubeflow application directory for this deployment.
+    export BASE_DIR=<path to a base directory>
```

Comment:
Is KF_NAME now also used in other cloud providers? not sure if we need one extra level? 
My guess for the new semantics is

${BASE_DIR}
 - kubeflow_aws.yaml

After kfctl apply 
${BASE_DIR}
 - kubeflow_aws.yaml
 - kustomize/
 - aws_configs/ (infrastructure configs)
 - app.yaml 

Since `kfctl delete` doesn't have an option to specify the dir, we have to come back to ${BASE_DIR} to run this command. `kfctl apply` specific config, it can run anywhere but all other configuration will be generated in the same folder as `kubeflow_config.yaml`

Reply:
@Jeffwan Thanks for your comments!

My understanding is that kfctl puts a copy of the config file in the directory in which it's run. (I adjusted the other cloud providers to take this into account - see PRs - but I didn't know if eksctl expected the config file to be in `tmp`. Based on your comments, I can now assume the AWS process is OK with the new location of the config file.)

So, after the user runs `kfctl apply`, the `KF_DIR` directory contains:

* `kubeflow_aws.yaml` (kfctl puts this here; the user does not need to do that)
* `kustomize/`
* `aws_config/`

There's no `app.yaml` any more.

We're standardising on being specific about `${BASE_DIR}` and `${KF_NAME}` -> `${KF_DIR}`, to reduce confusion about the deployment name and the directory path. See
https://github.com/kubeflow/website/issues/1252.

I've updated this PR to use the config file from the `KF_DIR` directory, as placed there by kfctl. I've also included instructions for the following workflow:

* Use `kfctl build` to set up the config.
* Customise the config files.
* Use `kfctl apply` to deploy.

Please would you verify that this will work for AWS?

Reply:
This looks good to me.

Reply:
Thanks!